### PR TITLE
Db modify

### PR DESCRIPTION
**Pull request by nully0x**
This PR made changes to db tables as referenced in issue #54

To reproduce: after cloning run `npx sequelize db:migrate` in  api folder

### CODE REVIEW

File: api/app/sequelize/migrations/20230703214456-modify_pr_url.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        type: Sequelize.ENUM('queued', 'not queued'),
+        defaultValue: 'queued'
+      }
+    );
+
+    //add email_address column to users table
+
+    await queryInterface.addColumn(
+      'users',
+      'email_address',
```
Comment (Extheoisah) [inline api/app/sequelize/migrations/20230703214456-modify_pr_url.js:34]: You need to add this field to the users model

File: api/app/sequelize/migrations/20230703214456-modify_pr_url.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        type: Sequelize.STRING,
+        unique: true,
+      }
+    );
+
+    //add jwt column to users table
+
+    await queryInterface.addColumn(
+      'users',
+      'jwt',
```
Comment (Extheoisah) [inline api/app/sequelize/migrations/20230703214456-modify_pr_url.js:45]: Same here -> add to the user model

**Review by Extheoisah — CHANGES_REQUESTED** [review]

File: api/app/sequelize/migrations/20230703214456-modify_pr_url.js (hunk 2 above)
Comment (nully0x) [reply api/app/sequelize/migrations/20230703214456-modify_pr_url.js:45]: Did you run it locally??

**Comment by Extheoisah** [comment]
closing this PR. the issue has been resolved by https://github.com/bitcointranscripts/transcription-review-backend/pull/56

**Pull request closed without merging**